mycollab-web/src/main/java/com/mycollab/module/user/accountsettings/team/view/UserListViewImpl.java

```diff
@@ -164,7 +164,7 @@ private Component generateMemberBlock(SimpleUser member) {
 
         if (RegisterStatusConstants.NOT_LOG_IN_YET.equals(member.getRegisterstatus())) {
             MButton resendBtn = new MButton(UserUIContext.getMessage(UserI18nEnum.ACTION_RESEND_INVITATION), clickEvent -> {
-                SendUserInvitationEvent invitationEvent = new SendUserInvitationEvent(member.getUsername(), "",
+                SendUserInvitationEvent invitationEvent = new SendUserInvitationEvent(member.getUsername(), member.getPassword(),
```

Comment:
If you debug the password value, then you know this password is encrypted and we don't know the original one. So the password (if any) we send to the user in this case can not be used to log in the system. 

If we resend the invitation, the user can get the password in the previous email. If they can't get this email, he can use the forgot password.